crates/sui-core/src/epoch/committee_store.rs

```diff
@@ -77,6 +77,16 @@ impl CommitteeStore {
             .1
     }
 
+    /// Return the committee specified by `epoch`. If `epoch` is `None`, return the latest committee.
+    pub fn get_or_latest_committee(&self, epoch: Option<EpochId>) -> SuiResult<Committee> {
```

Comment:
Iirc, in the latest explorer screens one could see the validators per epoch + who signed each checkpoint. Is this api related to that?

Reply:
Probably?